src/nvim/lua/base64.c

```diff
@@ -0,0 +1,65 @@
+#include <assert.h>
+#include <lauxlib.h>
+#include <lua.h>
+
+#include "nvim/base64.h"
+#include "nvim/lua/base64.h"
+#include "nvim/memory.h"
+
+static int nlua_base64_encode(lua_State *L)
```

Comment:
👍  the `L` convention is very common in Lua so we should change our other code to use that too.